adapters/spi/tomcat-adapter-spi/src/main/java/org/keycloak/adapters/tomcat/GenericPrincipalFactory.java

```diff
@@ -22,13 +22,7 @@
 
 import javax.security.auth.Subject;
 import java.security.Principal;
-import java.security.acl.Group;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Enumeration;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Can you please remove "start" imports and use the individual classes as was before? In general, we try to avoid star imports in our java classes.

Reply:
Done